phac-nml/irida-next
# Group samples workflow submission

## What does this PR do and why?
Added the ability for a user with at least analyst access to launch pipelines from the group samples table.
Fixes #455.

## Screenshots or screen recordings
![image](https://github.com/phac-nml/irida-next/assets/325703/217756bf-e006-4043-882a-de57ef255754)

## How to set up and validate locally
1. Login to irida-next.
2. Create a new project and a new sample within the new project.
3. Go back to project samples page and select the new sample.
4. Verify clicking the pipeline submission button works the same as on the project samples table.

Repeat with a user less than an analyst and verify the pipeline submission button is disabled.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
